src/backend/XmlHttpRequest.ts

```diff
@@ -396,7 +414,11 @@ export default class XmlHttpRequest extends BaseFileSystem implements FileSystem
   private _requestFileAsync(p: string, type: 'json', cb: BFSCallback<any>): void;
   private _requestFileAsync(p: string, type: string, cb: BFSCallback<any>): void;
   private _requestFileAsync(p: string, type: string, cb: BFSCallback<any>): void {
-    asyncDownloadFile(this.getXhrPath(p), type, cb);
+    if (this._useFetch) {
```

Comment:
No worries! I didn't think you were being sassy. I was genuinely wondering if there were more than two places that needed that conditional.